test/elixir_google_scraper_web/controllers/api/v1/keyword_controller_test.exs

```diff
@@ -0,0 +1,80 @@
+defmodule ElixirGoogleScraperWeb.Api.V1.KeywordControllerTest do
+  use ElixirGoogleScraperWeb.ConnCase, async: true
+
+  alias ExOauth2Provider.AccessTokens
+  alias ExOauth2Provider.Applications
+
+  describe "POST create/2" do
+    test "returns 201 status with empty response body when the token is valid", %{conn: conn} do
+      user = insert(:user)
+      attrs = %{name: "Application", redirect_uri: "https://example.org/endpoint"}
+      file = %Plug.Upload{content_type: "text/csv", path: "test/fixture/keywords.csv"}
+
+      {_, oauth_app} = Applications.create_application(nil, attrs, otp_app: :elixir_google_scraper)
+
+      {_, acess_token} =
+        AccessTokens.create_token(user, %{application: oauth_app}, otp_app: :elixir_google_scraper)
```

Comment:
Maybe later, we can create a helper for this 👍

Reply:
@rosle I did in here https://github.com/junan/elixir_google_scraper/pull/45/